### PR TITLE
File uploaded 

### PR DESCRIPTION
**Pull request by dayanandviveki**
All assignment uploaded with new branch

### CODE REVIEW

File: Dayanand Viveki/HTML/ASSI3/Q 2/MI-MEAN-9.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                <li>Bipin</li>
+                <li>Priyanka</li>
+                <li>Yeshaswi</li>
+            </ol>
+        </span>
+
+    </section>
+
+
+
```
Comment (sourabh-kanwade) [inline Dayanand Viveki/HTML/ASSI3/Q 2/MI-MEAN-9.html]: remove extra spaces in all files .

Comment (sourabh-kanwade) [reply Dayanand Viveki/HTML/ASSI3/Q 2/MI-MEAN-9.html]: .

File: Dayanand Viveki/HTML/ASSI1/ASSI1.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<html>
+
```
Comment (preetipie22) [inline Dayanand Viveki/HTML/ASSI1/ASSI1.html]: Hey @dayanandviveki , remove the extra lines here and in other codes

File: Dayanand Viveki/HTML/ASSI1/ASSI4.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        }
+        b{
+            font-weight: 900;
+            font-size: 30px;
+        }
+    </style>
+</head>
+<body>
+    <div>
+        <h3>I am <u><i>Dayanand Viveki</i></u>. I belong to <b>Ahmednagar, Maharashtra</b> but recently am shifted Karve Nagar,Pune, Maharashtra.
```
Comment (preetipie22) [inline Dayanand Viveki/HTML/ASSI1/ASSI4.html]: The <u> tag represents some text that is unarticulated and styled differently from normal text, such as misspelled words or proper names in Chinese text.
Instead use <ins> for inserted underlined text

File: Dayanand Viveki/HTML/ASSI2/ASSI1.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        <p style="font-size: 25px;">By some accounts, they disagreed about nearly everything during that first meeting, but by the following
+        year they struck a <span style="background-color: yellow;">partnership.</span>
+        Working from their dorm rooms, they built a search engine that used links to determine the importance of
+        individual pages on the <span style="color:blue">World Wide Web .</p>
+        <p style="font-size: 25px;">They called this search engine <span style="background-color: cyan;font-style: italic;">Backrub.</span></p>
+
+        <hr style="height: 3px;background-color: gray;">
+        <p style="font-size: 35px;font-weight: bold;">Initial public offering</p>
+        <p style="font-size: 25px;">Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+        At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for<span style="text-decoration: line-through;"> 30</span> <u>20</u> years, until the year 2024.</p>
```
Comment (preetipie22) [inline Dayanand Viveki/HTML/ASSI2/ASSI1.html]: Use `<ins>` instead of `<u>`

**Review by preetipie22 — CHANGES_REQUESTED** [review]
Please make the changes

File: Dayanand Viveki/HTML/ASSI3/Q 2/MI-MEAN-9.html (hunk 1 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/ASSI3/Q 2/MI-MEAN-9.html]: ok

File: Dayanand Viveki/HTML/ASSI1/ASSI4.html (hunk 3 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/ASSI1/ASSI4.html]: ok mdm

File: Dayanand Viveki/HTML/ASSI2/ASSI1.html (hunk 4 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/ASSI2/ASSI1.html]: ok mdm

File: Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                            <li>Dipti</li>
+                        </ul>
+                    </li>
+                    <li>SYSTEM-2
+                        <ul>
+                            <li>Triveni</li>
+                            <li>Kalyani</li>
+                        </ul>
+                    </li>
+
```
Comment (preetipie22) [inline Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html]: remove extra lines

File: Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+                </ol>
+
```
Comment (preetipie22) [inline Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html]: here also

**Review by sourabh-kanwade — APPROVED** [review]

**Review by preetipie22 — DISMISSED** [review]
Remove extra lines

File: Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html (hunk 5 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/ASSI3/Q 1/ASSI1.html]: ok mdm

**Comment by avinash0070** [comment]
Change title give proper name 
and mention in your title which files you added

**Comment by dayanandviveki** [comment]
dublicate file created

**Pull request closed without merging**